Delivered-To: daniel@ffwll.ch
Received: by 10.43.162.137 with SMTP id mk9csp324914icc;
        Tue, 7 Oct 2014 08:14:52 -0700 (PDT)
X-Received: by 10.68.57.168 with SMTP id j8mr4239253pbq.106.1412694892029;
        Tue, 07 Oct 2014 08:14:52 -0700 (PDT)
Return-Path: <intel-gfx-bounces@lists.freedesktop.org>
Received: from gabe.freedesktop.org (gabe.freedesktop.org. [131.252.210.177])
        by mx.google.com with ESMTP id oe6si7115142pbc.156.2014.10.07.08.14.51
        for <daniel@ffwll.ch>;
        Tue, 07 Oct 2014 08:14:52 -0700 (PDT)
Received-SPF: none (google.com: intel-gfx-bounces@lists.freedesktop.org does not designate permitted sender hosts) client-ip=131.252.210.177;
Authentication-Results: mx.google.com;
       spf=neutral (google.com: intel-gfx-bounces@lists.freedesktop.org does not designate permitted sender hosts) smtp.mail=intel-gfx-bounces@lists.freedesktop.org
Received: from gabe.freedesktop.org (localhost [127.0.0.1])
	by gabe.freedesktop.org (Postfix) with ESMTP id 5000089DA5;
	Tue,  7 Oct 2014 08:14:51 -0700 (PDT)
X-Original-To: intel-gfx@lists.freedesktop.org
Delivered-To: intel-gfx@lists.freedesktop.org
Received: from mga03.intel.com (mga03.intel.com [134.134.136.65])
 by gabe.freedesktop.org (Postfix) with ESMTP id A414589DA5
 for <intel-gfx@lists.freedesktop.org>; Tue,  7 Oct 2014 08:14:49 -0700 (PDT)
Received: from orsmga001.jf.intel.com ([10.7.209.18])
 by orsmga103.jf.intel.com with ESMTP; 07 Oct 2014 08:11:02 -0700
X-ExtLoop1: 1
X-IronPort-AV: E=Sophos;i="5.04,670,1406617200"; d="scan'208";a="584799260"
Received: from rosetta.fi.intel.com (HELO rosetta) ([10.237.72.93])
 by orsmga001.jf.intel.com with ESMTP; 07 Oct 2014 08:13:35 -0700
Received: by rosetta (Postfix, from userid 1000)
 id 9FEE080052; Tue,  7 Oct 2014 18:15:03 +0300 (EEST)
From: Mika Kuoppala <mika.kuoppala@linux.intel.com>
To: intel-gfx@lists.freedesktop.org
Date: Tue,  7 Oct 2014 18:15:01 +0300
Message-Id: <1412694901-2982-1-git-send-email-mika.kuoppala@intel.com>
X-Mailer: git-send-email 1.9.1
Subject: [Intel-gfx] [PATCH] drm/i915: remove bogus intel_increase_pllclock
X-BeenThere: intel-gfx@lists.freedesktop.org
X-Mailman-Version: 2.1.15
Precedence: list
List-Id: Intel graphics driver community testing & development
 <intel-gfx.lists.freedesktop.org>
List-Unsubscribe: <http://lists.freedesktop.org/mailman/options/intel-gfx>,
 <mailto:intel-gfx-request@lists.freedesktop.org?subject=unsubscribe>
List-Archive: <http://lists.freedesktop.org/archives/intel-gfx>
List-Post: <mailto:intel-gfx@lists.freedesktop.org>
List-Help: <mailto:intel-gfx-request@lists.freedesktop.org?subject=help>
List-Subscribe: <http://lists.freedesktop.org/mailman/listinfo/intel-gfx>,
 <mailto:intel-gfx-request@lists.freedesktop.org?subject=subscribe>
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Errors-To: intel-gfx-bounces@lists.freedesktop.org
Sender: "Intel-gfx" <intel-gfx-bounces@lists.freedesktop.org>

commit b680c37a4d145cf4d8f2b24e46b1163e5ceb1d35
Author: Daniel Vetter <daniel.vetter@ffwll.ch>
Date:   Fri Sep 19 18:27:27 2014 +0200

    drm/i915: DocBook integration for frontbuffer tracking

Moved the code to intel_frontbuffer.c but this bit
got stranded.

Signed-off-by: Mika Kuoppala <mika.kuoppala@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 1363a93..cbe2a5d 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -73,8 +73,6 @@ static const uint32_t intel_cursor_formats[] = {
 	DRM_FORMAT_ARGB8888,
 };
 
-static void intel_increase_pllclock(struct drm_device *dev,
-				    enum pipe pipe);
 static void intel_crtc_update_cursor(struct drm_crtc *crtc, bool on);
 
 static void i9xx_crtc_clock_get(struct intel_crtc *crtc,
-- 
1.9.1

_______________________________________________
Intel-gfx mailing list
Intel-gfx@lists.freedesktop.org
http://lists.freedesktop.org/mailman/listinfo/intel-gfx
